### PR TITLE
[DOC-333] [2024.1] Added support bundle phase2

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/yugabyte-platform/troubleshoot/universe-issues/#use-support-bundles

### CODE REVIEW

**Review by Sahith02 — COMMENTED** [review]
Overall changes look good, left a few comments

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 1]
```diff
@@ -120,6 +124,48 @@ You can create a support bundle as follows:
 
   The **Support Bundles** dialog allows you to either download the bundle or delete it if it is no longer needed. By default, bundles expire after ten days to free up space.
 
+### Support bundle components
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:127]: oh yes for sure. Even though the rest seem pretty self-explanatory, its good to provide some detail. Ill add them. Thank you! @Sahith02

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 2]
```diff
@@ -166,6 +156,84 @@ support_bundle/
...
+
+YBA also provides a runtime flag `yb.support_bundle.allow_cores_collection`, which is used to globally disable cores collection across any new support bundles generated on the platform. This flag can only be set by the SuperAdmin and is true by default.
+
+#### YB-Controller logs
+
+YBC logs generated in `/controller/logs` folder in the YugabyteDB nodes (if YB-Controller is enabled).
+
+#### Kubernetes info
+
+This component exclusively includes metadata for Kubernetes-based universes. The detailed metadata list is as follows:
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:212]: We should also add a note here saying that these below files are only collected by YBA if we have sufficient permissions to get this info. Else the file collection is skipped.

**Review by Sahith02 — APPROVED** [review]
Left one comment. Rest looks good, thanks!

**Comment by aishwarya24** [comment]
Created PR - https://github.com/yugabyte/yugabyte-db/pull/22570

**Pull request closed without merging**